ciao-launcher/main.go

```diff
@@ -356,6 +353,28 @@ DONE:
 			if err != nil {
 				break DONE
 			}
+			clusterConfig,err := client.conn.ClusterConfiguration()
+			if err != nil {
+				glog.Errorf("Unable to get Cluster Configuration %v", err)
+				dialCh <- err
+				break DONE
+			}
+			computeNet = clusterConfig.Configure.Launcher.ComputeNetwork
+			mgmtNet = clusterConfig.Configure.Launcher.ManagementNetwork
+			diskLimit = clusterConfig.Configure.Launcher.DiskLimit
+			memLimit = clusterConfig.Configure.Launcher.MemoryLimit
+			if networking.Enabled() {
+				ctx, _ := context.WithCancel(context.Background())
+				ch := initNetworking(ctx)
+				err := <-ch
+				if err != nil {
+					glog.Errorf("Failed to init network: %v\n", err)
+					break DONE
+				}
+			}
+			defer shutdownNetwork()
+
+			ovsCh = startOverseer(&wg, client)
```

Comment:
@markdryan, not sure what happened by I'm getting 14, is that ok, or should we still split it?
14 main connectToServer ciao-launcher/main.go:316:1